stix_shifter_modules/guardium/stix_transmission/api_client.py

```diff
@@ -1,250 +1,73 @@
 from stix_shifter_utils.stix_transmission.utils.RestApiClient import RestApiClient
```

Comment:
`RestApiClient` is not used anymore in this api_client. you should remove this import

Reply:
done